lblod/ember-rdfa-editor
# Display new image icon if host app has `@appuniversum/ember-appuniversum` >= 2.16.0 installed

### Overview
This PR updates the icon show in the image-insert menu-button if the host-app consuming this addon has `@appuniversum/ember-appuniversum` >= 2.16.0.

If the version of the `@appuniversum/ember-appuniversum`  installed in the host app is:
- \>= 2.16.0, the `image` icon is displayed
- < 2.16.0, the `website` icon is displayed

This addon checks for the installed version of `@appuniversum/ember-appuniversum` using the `@embroider/macros` package.

##### connected issues and PRs:




### Setup



### How to test/reproduce
- Open up the test-app (which has `@appuniversum/ember-appuniversum` 2.16.0 installed)
- Check if the icon displayed in the image-insert menu is the `image` icon
- Lower the installed version of `@appuniversum/ember-appuniversum` to below 2.16.0
- Check if the icon displayed in the image-insert menu is the `website` icon

### Checks PR readiness
